inspirehep/dojson/hep/fields/bd6xx.py

```diff
@@ -145,10 +146,16 @@ def _freekey_get_dict(elem):
 
     def _get_keyword_dict(key, elem):
         if _is_thesaurus(key) and elem.get('a'):
+            _schema = load_schema('hep')
+            valid_keywords = _schema['properties']['keywords']['items']['properties']['schema']['enum']
+            if elem.get('2').upper() in valid_keywords:
+                schema = elem.get('2').upper()
+            else:
+                schema = ''
```

Comment:
This can be written as in:

```python
schema_in_marc = elem.get('2').upper()
schema = schema_in_marc if schema_in_marc in valid_keywords else ''
```